pipelines/e2e/e2e.yaml

```diff
@@ -36,6 +36,12 @@ spec:
       type: string
       description: Public key to use for validation by the enterprise contract
       default: k8s://openshift-pipelines/public-key
+    - name: enterpriseContractExtraRuleData
+      type: string
+      description:
```

Comment:
I believe this should be `description: |` if you are going to do a multiline string

Same throughout

Reply:
@johnbieren Added, thanks. Not sure why the commit message lint is failing though...

Reply:
Because it isn't following conventional commits. It should be like `feat: support new EXTRA_RULE_DATA parameter`.. `verify-enterprise-contract` isn't a proper prefix

Reply:
@johnbieren I reworded all my commits with proper prefixes (`feat`, `chore`, and `docs`) but it's still complaining. Time to pull out a regex tool!

Reply:
2 of the 3 commits start with a capital letter after `feat:` that is not allowed

Reply:
aha, that would do it - looks good now, thanks!